### PR TITLE
Login and register

### PR DESCRIPTION
**Pull request by Ahmadsaniliman**
Created the login and register page ui.

### CODE REVIEW

File: lib/features/onboarding/widget/login_with_card.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTapped,
+      child: Container(
+        height: 44.h,
+        width: 60.w,
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(10),
+            color: const Color(0xFFECECEC)),
```
Comment (codeswot) [inline lib/features/onboarding/widget/login_with_card.dart]: Move this to a class in the theme folder
Like theme/colors/app_colors.dart
And any other color you may be using

File: lib/shared/widgets/regiser_login_button.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class LoginRegisterButton extends StatelessWidget {
```
Comment (codeswot) [inline lib/shared/widgets/regiser_login_button.dart]: I believe we have a primary button? We can use the primary button but expose any property that may be different and just use it

File: lib/features/onboarding/widget/login_with_card.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTapped,
+      child: Container(
+        height: 44.h,
+        width: 60.w,
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(10),
```
Comment (codeswot) [inline lib/features/onboarding/widget/login_with_card.dart]: Use the .r for radius

File: lib/features/onboarding/widget/custom_text_form_field.dart [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        decoration: InputDecoration(
+          contentPadding: EdgeInsets.symmetric(
+            vertical: 20.h,
+            horizontal: 15.w,
+          ),
+          border: InputBorder.none,
+          focusedBorder: OutlineInputBorder(
+            borderRadius: BorderRadius.circular(10),
+            borderSide: const BorderSide(
+              width: 2,
```
Comment (codeswot) [inline lib/features/onboarding/widget/custom_text_form_field.dart]: Use .w here

File: lib/features/onboarding/pages/register.dart [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+          left: -25.w,
+          child: const AppImageWidget(
+            reference: AppAssets.authdownline2,
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.only(top: 90.h, left: 25.w, right: 25.w),
+          child: SizedBox(
+            height: double.infinity.h,
+            width: double.infinity.w,
```
Comment (codeswot) [inline lib/features/onboarding/pages/register.dart]: Use 1.sw or 1.sh if you want full decide width or height, or you can just simple use double.infinity without .h or .w

File: lib/features/onboarding/pages/register.dart [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+                            children: [
+                              LoginWithCard(
+                                image: AppAssets.googleIcon,
+                                onTapped: () {},
+                              ),
+                              LoginWithCard(
+                                image: AppAssets.facebookIcon,
+                                onTapped: () {},
+                              ),
+                              LoginWithCard(
```
Comment (codeswot) [inline lib/features/onboarding/pages/register.dart]: Rename to SSOLoginCard

**Review by codeswot — CHANGES_REQUESTED** [review]

File: lib/shared/widgets/regiser_login_button.dart (hunk 2 above)
Comment (Ahmadsaniliman) [reply lib/shared/widgets/regiser_login_button.dart]: ok sir , i  done all the changes

**Review by codeswot — APPROVED** [review]

**Pull request merged**